### ISSUE TITLE
Implementar perguntas anónimas (pendentes de aprovação)

### ISSUE THREAD

**Issue by IvoPereira** [devpt-org/welcome-bot#45]
Seria interessante termos uma funcionalidade em que um utilizador através de:
- uma mensagem direta para o bot (se possível) OU
- através de uma página web básica com um formulário OU
- num canal em que só o próprio utilizador pode ver (costumava-se usar algo semelhante para sistemas de tickets no Discord?)

Assim que esta mensagem é recebida pelo bot, esta fica pendente de aprovação num canal apenas visível para admins (sempre anonimificada).

**Exemplo:**

> **devPT Bot says:**
> Mensagem 09e0c047-bf50-4983-a245-08c6cb2654f1
> "Preciso de procurar outras oportunidades que paguem mais mas não sei onde. Alguém me pode sugerir bons sites para encontrar trabalho?"

Assim que aprovada (p.ex. um membro do staff respondendo neste canal privado com um `!aprovar 09e0c047-bf50-4983-a245-08c6cb2654f1`) esta é publicada num canal público a designar - se possível utilizando a nova funcionalidade de Fóruns.

Assim, qualquer utilizador que queira manter o seu anonimato em perguntas mais pessoais poderá fazê-lo sem problema.

**Comment by gustavos00** [comment]
Podia fazer-se algo estilo tickets (como tu mencionaste) ou threads. Digo threads porque assumindo que o utilizador vá demorar 2 minutos a fazer a sua questão, ninguém ficará a espera de ver novas threads. 
Ao enviar a mensagem, o bot guarda a mensagem e já envia a mensagem para vós. 
Sobre o forúm, é ver que a dependência do Discord.JS já aceita esse tipo de abordagens.

**Comment by zorkpt** [comment]
on it :)

**Comment by C0ffeeL0ver** [comment]
@zorkpt  foi feito?

**Comment by Alf0nso** [comment]
@C0ffeeL0ver penso que nunca se chegou a implementar não... terias interesse em tocar? Ou perguntas porque gostarias de utilizar semelhante ferramenta?

**Comment by C0ffeeL0ver** [comment]
Estava a pensar em algo assim:

![image_2024-09-05_164743682](https://github.com/user-attachments/assets/e08b91ac-2996-4c95-9d76-f8054b09d9ab)

**Comment by Alf0nso** [comment]
@C0ffeeL0ver A estrutura parece-me bastante decente, mas não sei se colocaria um auto delete, não? Se é uma pergunta anónima que pode servir de informação para futuros membros do discord então não sei a que ponto não tem mais sentido preservar

o "Open the chat to everyone answer" acho que seria mais "Open the chat so that anyone can answer" (sou piquinhas eu sei, desculpa)

**Comment by C0ffeeL0ver** [comment]
se for para preservar creio que o mais indicado seja o forum posts como está feito no canal dos classificados

**Comment by Alf0nso** [comment]
@C0ffeeL0ver Sim, acho que efectivamente usar um forum seria o melhor, mas também não sei o que opinas.. ou seja eu acho que seria a abordagem mais limpa

**Comment by C0ffeeL0ver** [comment]
@Alf0nso A minha ideia inicial era o estilo tickets como referido em cima, mas como o objetivo é preservar a informação a utilização do forum é a abordagem mais limpa.
Além disso, no forum é possível a utilização e filtração por tags, o admin que aprovar as mensagens pode atribuir uma tag as mesmas.

**Comment by Alf0nso** [comment]
@C0ffeeL0ver parece-me uma muito boa ideia :) overall gosto! (desculpa não ter respondido mais cedo)
